tests/basic.bats

```diff
@@ -3,93 +3,93 @@
 load helpers
 
 @test "from" {
-	cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json --image alpine)
-	buildah delete --name=$cid
-	cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json         alpine)
-	buildah delete --name=$cid
-	cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json         alpine  i-love-naming-things)
-	buildah delete --name=i-love-naming-things
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json --image alpine)
+  buildah delete $cid
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json         alpine)
+  buildah delete $cid
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json         alpine  i-love-naming-things)
+  buildah delete i-love-naming-things
 }
 
 @test "from-defaultpull" {
-	cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json --image alpine)
-	buildah delete        $cid
+  cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json --image alpine)
+  buildah delete        $cid
 }
 
 @test "from-nopull" {
-	run buildah from --pull=false --signature-policy ${TESTSDIR}/policy.json --image alpine
-	[ "$status" -eq 1 ]
+  run buildah from --pull=false --signature-policy ${TESTSDIR}/policy.json --image alpine
+  [ "$status" -eq 1 ]
 }
```

Comment:
I'm not particularly attached to them, but if the patch isn't already making non-whitespace changes to the same lines, it just makes the patch longer and a bit harder to review.